src/sql/workbench/contrib/queryplan2/browser/media/queryPlan2.css

```diff
@@ -0,0 +1,71 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.actionbar-container {
```

Comment:
Please scope this better - actionbar-container seems too generic 

(same for other ones in this file such as properties-container,  prop-table-header, etc)